assets/helpers/internationalisation/countryGroup.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+
+// ----- Imports ----- //
+
+import * as cookie from 'helpers/cookie';
+import { getQueryParameter } from 'helpers/url';
+
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
+
+export type CountryGroupId = 'GBPCountries' | 'UnitedStates' | 'AUDCountries' | 'EURCountries' | 'International';
+
+export type CountryGroup = {
+  name: string,
+  currency: IsoCurrency,
+  countries: IsoCountry[],
+  supportInternationalizationId: string,
+};
+
+type CountryGroups = {
+  [CountryGroupId]: CountryGroup
+}
+
+const countryGroups: CountryGroups = {
+  GBPCountries: {
+    name: 'United Kingdom',
+    currency: 'GBP',
+    countries: ['GB', 'FK', 'GI', 'GG', 'IM', 'JE', 'SH'],
```

Comment:
As discussed - it could maybe be worth trying to read these from the library itself by outputting some JSON in the Twirl template. That way we don't have to keep the Scala library and this code in sync.

But equally, this stuff doesn't change often and this way is arguably clearer and more explicit. So if we have some prominent comments reminding people that this has to match the Scala library then I'm happy

Reply:
If we create it as a [JavaScript] library, maybe the code could also sit alongside its Scala cousin: https://github.com/guardian/support-internationalisation